dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/ws/security/kerberos/Krb5HelperJdk8.java

```diff
@@ -56,8 +55,9 @@
     /**
      * We don't do anything with the process, but having it set allows us to only be activated by DS if criteria we set
      * about the Java version are met.
+     * IBM JDK 1.8 has value added and the vendor name can be IBM or not IBM
      */
-    @Reference(policy = ReferencePolicy.STATIC, target = "(&(java.specification.version>=1.8)(java.vendor=ibm corporation))")
+    @Reference(policy = ReferencePolicy.STATIC, target = "(&(java.specification.version>=1.8)(java.specification.version<=9.0)(java.vendor=ibm corporation))")
```

Comment:
We only support Java 8 and higher so the beginning condition isn't needed.  <=9 seems wrong.  If anything it could be < 9, but why not just do =1.8.  That is the easiest conditional.  Like I said in the issue:
(&(java.specification.version=1.8)(java.vendor=ibm corporation))